src/buildApplication/buildApplication.ts

```diff
@@ -80,10 +82,14 @@ async function prepareAndBuildWorker(
 		return;
 	}
 
+	// NOTE: Middleware manifest logic will be removed in the new routing system.
```

Comment:
sorry for being nitpicky but wherever you have these comments could you please also mention your issue?
```suggestion
	// NOTE: Middleware manifest logic will be removed in the new routing system (see issue #129).
```
sorry I am sure you're going to clean everything up but it's just to be extra extra sure everything gets cleaned up (and in the eventuality something is left behind later we can see what "new routing system" is referring to and know that it can be cleaned up)